vtexdocs/devportal
# chore: update rapidoc submodule

#### What is the purpose of this pull request?

To fix the padding in the code container in API Reference pages.

#### What problem is this solving?

The padding was to large and hid a significant part of the code.

#### How should this be manually tested?

Open any API Reference page and check if the padding and tab size in the code preview are a adequate size.

#### Screenshots or example usage

| Before | After |
| --- | --- |
| ![image](https://github.com/vtexdocs/devportal/assets/62757720/158a6865-2ef9-4147-a7bb-4b82f26291f3) | ![image](https://github.com/vtexdocs/devportal/assets/62757720/e6ee58e1-5c63-4555-835b-efbb679ed056) | 

#### Types of changes
